.jenkins/pytorch/test.sh

```diff
@@ -7,6 +7,9 @@
 # shellcheck disable=SC2034
 COMPACT_JOB_NAME="${BUILD_ENVIRONMENT}"
 
+# Get fully qualified path using realpath
+CUSTOM_TEST_ARTIFACT_BUILD_DIR=$(realpath ${CUSTOM_TEST_ARTIFACT_BUILD_DIR:-${PWD}/../})
```

Comment:
nit: why do we need this default back to `${PWD}/../` behavior? can we remove it entirely?

Reply:
I wanted to keep them so that the circleci builds don't actually change their current behavior.

Trying to make the least obstructive change here.